source/CMakeLists.txt

```diff
@@ -4,18 +4,18 @@
 # LICENSE.txt file in the root directory of this source tree.
 
 add_library(unifex "")
-
 target_sources(unifex
   PRIVATE
     inplace_stop_token.cpp
     manual_event_loop.cpp
     trampoline_scheduler.cpp
     thread_unsafe_event_loop.cpp
     timed_single_thread_context.cpp)
+target_link_libraries(unifex PUBLIC unifex-interface)
 
 if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
   target_sources(unifex
-    PRIVATE
+    INTERFACE
```

Comment:
I can't think of a reason for this change ?

Reply:
Probably a holdover from a time when I naively thought simply making that target INTERFACE would cause features to propagate to dependencies. I can revert that.